### PR TITLE
Chore(ci): Modify label mechanism by only parsing result*.xml

### PR DESCRIPTION
**Pull request by jingh8**
Experimenting new label mechanism by only parsing result*.xml

### CODE REVIEW

File: vars/testHelper.groovy [hunk 1]
```diff
@@ -45,7 +45,7 @@ def runIntegrationTests(String namespace, String service, String testedEnv, Stri
         sh "/bin/rm -rf output/ || true"
         sh "mkdir output"
         testResult = null
-        List<String> failedTestSuites = [];
+        //List<String> failedTestSuites = [];
```
Comment (themarcelor) [inline vars/testHelper.groovy:48]: Should we remove the commented out legacy code?

File: vars/testHelper.groovy [hunk 2]
```diff
@@ -56,8 +56,8 @@ def runIntegrationTests(String namespace, String service, String testedEnv, Stri
           }
         }
         // check XMLs inside the output folder
-        failedTestSuites = xmlHelper.identifyFailedTestSuites()
-        def featureLabelMap = xmlHelper.assembleFeatureLabelMap(failedTestSuites)
+        //failedTestSuites = xmlHelper.identifyFailedTestSuites()
```
Comment (themarcelor) [inline vars/testHelper.groovy:59]: Should we remove the commented out legacy code?

Comment (jingh8) [reply vars/testHelper.groovy]: Done

**Review by themarcelor — APPROVED** [review]
+1

**Pull request merged**